src/main/java/company/Company.java

```diff
@@ -0,0 +1,114 @@
+    private List<Employee> candidateList = new ArrayList<>();
+    private List<Employee> filteredEmployeeList = new ArrayList<>();
+
+    /**
+     * Fills {@link Company#getCandidateList()} with random data of {@link Employee}
+     */
+    public Company() {
+        Faker faker = new Faker();
+        candidateList.add(new QA(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new QA(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new QA(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new QA(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+
+        candidateList.add(new Director(faker.name().firstName(), faker.random().nextInt(500, 3000),true));
+        candidateList.add(new Director(faker.name().firstName(), faker.random().nextInt(500, 3000),true));
+        candidateList.add(new Director(faker.name().firstName(), faker.random().nextInt(500, 3000),true));
+
+        candidateList.add(new Developer(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Developer(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Developer(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Developer(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+
+        candidateList.add(new Manager(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Manager(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Manager(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Manager(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+
+        candidateList.add(new Accountant(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Accountant(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Accountant(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+        candidateList.add(new Accountant(faker.name().firstName(), faker.random().nextInt(500, 3000), faker.random().nextBoolean()));
+    }
+
+    /**
+     * @return employeeList with {@link Employee}
+     * @throws EmployeeNotFoundException
+     */
+    public List<Employee> getEmployeeList() throws EmployeeNotFoundException {
```

Comment:
throws EmployeeNotFoundException -> it's redundant in case u're using throw within the method